gcloud-java-examples/README.md

```diff
@@ -33,7 +33,7 @@ To run examples from your command line:
 
 1. Login using gcloud SDK (`gcloud auth login` in command line)
 
-2. Set your current project using `gcloud config set project PROJECT_ID`
+2. If you are running an example other than `ResourceManagerExample`, set your current project using `gcloud config set project PROJECT_ID`. If you are running `ResourceManagerExample`, unset your project by using `gcloud config unset project`.
```

Comment:
I initially thought it would be necessary because the Cloud Resource Manager API is by default turned off for projects, but it works even when I specify a project that doesn't have the API enabled.  I'll change the wording.